lib/src/license_detection/license.dart

```diff
@@ -31,6 +33,46 @@ class License {
   }
 }
 
+@sealed
+class Ngram {
```

Comment:
```suggestion
class NGram {
```

The word is `n-gram`

See: https://dart.dev/guides/language/effective-dart/style#do-capitalize-acronyms-and-abbreviations-longer-than-two-letters-like-words